### PR TITLE
Fixes reusable epipens

### LINKED ISSUE #24154: You can put reagents into used medipens
And medipens pierce things that are nopierce
lost a nuke ops round to this thanks a lot

Bug because medipens are deliberately made to be oneuse

### PR DESCRIPTION
**Pull request by ghost**
:cl: Xhuis
fix: Medipens are no longer reusable.
/:cl:

Closes #24154.

### CODE REVIEW

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 1]
```diff
@@ -81,6 +81,10 @@
 		user << "<span class='warning'>[src] is empty!</span>"
 		return
 	..()
+	if(!iscyborg(user))
+		desc = "[desc] It's been used up."
```
Comment (coiax) [inline code/modules/reagents/reagent_containers/hypospray.dm]: The description isn't needed, it's visible on examine how many units in a reagent container there are.

Comment (Xhuis) [inline code/modules/reagents/reagent_containers/hypospray.dm]: Even in a fix I manage to fuck something up goddamn

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 2]
```diff
@@ -81,6 +81,9 @@
 		user << "<span class='warning'>[src] is empty!</span>"
 		return
 	..()
+	if(!iscyborg(user))
+		reagents.maximum_volume = 0 //Makes them useless afterwards
+		container_type = 0
 	update_icon()
 	spawn(80)
```
Comment (coiax) [inline code/modules/reagents/reagent_containers/hypospray.dm]: Y NO ADDTIMER

Comment (Xhuis) [inline code/modules/reagents/reagent_containers/hypospray.dm]: Legacy code

**Pull request merged**